spec/api/v1/product_show_controller.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails-helper'
+
+RSpec.describe Api::V1::ProductShowController, type: :controller do
+  
+end
```

Comment:
This is currently empty, but we should continue to fill it out with tests for `GET#Show` to ensure it is returning everything expected in `Api::V1::ProductsController#show`.